examples/js/loaders/STLLoader.js

```diff
@@ -222,7 +222,7 @@ THREE.STLLoader.prototype = {
 
 			}
 
-			patternVertex = /vertex[\s]+([\-+]?[0-9]+\.?[0-9]*([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]+([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]+([eE][\-+]?[0-9]+)?)+/g;
+			patternVertex = /vertex[\s]+([\-+]?[0-9]+\.?[0-9]*([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]*([eE][\-+]?[0-9]+)?)+[\s]+([\-+]?[0-9]*\.?[0-9]*([eE][\-+]?[0-9]+)?)+/g;
```

Comment:
I think these regex changes make sense.

On this occasion some additional questions: 

1. The first position marked in my screenshot (where I added line breaks only for readability) is different in the first line from the second and third line's version. If writing a float like .3 instead of 0.3 is allowed in the STL file format, it should also be `*` in this place.

2. I wonder why there are `+` signs at the end of each line? I think they should be removed?

3. It would be simpler and less error-prone to write only one of these "lines" (in parentheses) and add `{3}`.
![regex](https://cloud.githubusercontent.com/assets/5700294/25666912/2e75693a-3022-11e7-938a-7cbfec17150b.JPG)

The same situation with the normals pattern.